geodata/models/raster/reader.py

```diff
@@ -109,6 +109,14 @@ def _read_files(self):
             self.raster_entry.creator = self.rfe.creator
             self.raster_entry.modifier = self.rfe.modifier
 
+            # There is a potential conflict between rasterio and whatever GDAL
+            # is available.  Rastio has an older form of GDAL and conflicts
+            # with a system GDAL if the version is different.  So far, the only
+            # issue seems to be with rastio's <source>.crs.  We can work around
+            # this by using GDAL directly.
+            gsrc = gdal.Open(str(file_path))
+            spatial_ref_wkt = gsrc.GetSpatialRef().ExportToWkt()
+            spatial_ref = SpatialReference(spatial_ref_wkt)
```

Comment:
@manthey, this change will conflict with my work, but I can easily resolve it in #109